src/linear_algebra/trace.lean

```diff
@@ -64,45 +64,36 @@ calc  matrix.trace ι R R (linear_map.to_matrix b b f)
 
 open_locale classical
 
-theorem trace_aux_reindex_range [nontrivial R] : trace_aux R b.reindex_range = trace_aux R b :=
```

Comment:
This is exactly the same as `trace_aux_eq` above and not needed any more below, so I deleted it.